src/main/kotlin/org/jitsi/rtp/rtcp/RtcpXrPacket.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.rtp.rtcp
+
+/**
+ * TODO: This is a stub implementation, so we can parse and ignore these packets.
```

Comment:
is there a subsequent pr that adds some special handling for xrs or can we forget about these?

Reply:
I don't have a PR right now. This just fixes the parsing, so we don't throw an exception. Instead, jmt logs a single line somewhere that it saw an RtcpXrPacket and it didn't know what to do with it (so it dropped it)